Makefile

```diff
@@ -0,0 +1,5 @@
+gen-expected:
+	helm3 template --namespace=default --values=tests/mapserver.yaml mapserver . > tests/mapserver-expected.yaml
```

Comment:
it would be better to have `HELM != helm3` at the beginning of Makefile just in case